docs/helm.md

```diff
@@ -64,3 +64,36 @@ spec:
 * `{{ .Values.resource.spec.name }}` would return "Nemo"
 * `{{ .Values.resource.spec.from }}` would return "Finding Nemo"
 * `{{ .Values.resource.spec.by }}` would return "Disney"
+
+### Using custom repo for charts
+
+For Lostrómos to be able to pull the charts from a remote repository, the remote
+repository should be pre-configured in the Lostrómos pod (one option is to add it
+using ```helm repo add```).
+
+```sh
+helm repo add foonemo https://username:password@repo.example.com
+```
+
+```foonemo``` is the repo name.
+
+Add chart entry to the spec, using the same repo name
+that was used to add, for the chart to be downloaded from the remote repo.<br/>
+Chart entry format: ```<repo name>/<chart name>:[semver]```
+
+```yaml
+
+apiVersion: stable.foobar.io/v1
+kind: Character
+metadata:
+  name: nemo
+spec:
+  Name: Nemo
+  From: Finding Nemo
+  By: Disney
+  Chart: foonemo/helloworld:1.2r345
```

Comment:
One thing we have always maintained for Lostrómos is the ability to run multiple instances against the same CRD, but have different outcomes. Hard coding this value to just use chart takes that away. 

To allow for this, we should use an annotation. It could default to `wpengine.com/lostromos.chart` but be changed via a config option.

Reply:
I like this idea, it also feels cleaner not to have the chart definition passed as a value to the chart.

Reply:
This change is in progress.

Reply:
done!